tests/func/test_update.py

```diff
@@ -71,3 +73,33 @@ def test_update_import_url(tmp_dir, dvc, tmp_path_factory):
 
     assert dst.is_file()
     assert dst.read_text() == "updated file content"
+
+
+def test_update_rev(tmp_dir, dvc, erepo_dir, monkeypatch):
+    with monkeypatch.context() as m:
```

Comment:
@ilgooz , if the reason for using `monkeypatch` is to `chdir` to `erepo_dir`, you can use instead:

```suggestion
    with erepo_dir.chdir():
```